docs/reference/configuration.md

```diff
@@ -0,0 +1,1067 @@
+| --- | --- | --- | --- |
+| `ELASTIC_APM_SERVICE_NAME` | `SERVICE_NAME` | `unknown-python-service` | `my-app` |
+
+The name of your service. This is used to keep all the errors and transactions of your service together and is the primary filter in the Elastic APM user interface.
+
+While a default is provided, it is essential that you override this default with something more descriptive and unique across your infrastructure.
+
+::::{note}
+The service name must conform to this regular expression: `^[a-zA-Z0-9 _-]+$`. In other words, the service name must only contain characters from the ASCII alphabet, numbers, dashes, underscores, and spaces. It cannot be an empty string or whitespace-only.
+::::
+
+
+
+### `server_url` [config-server-url]
+
+| Environment | Django/Flask | Default |
+| --- | --- | --- |
+| `ELASTIC_APM_SERVER_URL` | `SERVER_URL` | `'http://127.0.0.1:8200'` |
+
+The URL for your APM Server. The URL must be fully qualified, including protocol (`http` or `https`) and port. Note: Do not set this if you are using APM in an AWS lambda function. APM Agents are designed to proxy their calls to the APM Server through the lambda extension. Instead, set `ELASTIC_APM_LAMBDA_APM_SERVER`. For more info, see [AWS Lambda](/reference/lambda-support.md).
+
+
+## `enabled` [config-enabled]
+
+| Environment | Django/Flask | Default |
+| --- | --- | --- |
+| `ELASTIC_APM_ENABLED` | `ENABLED` | `true` |
+
+Enable or disable the agent. When set to false, the agent will not collect any data or start any background threads.
+
+
+## `recording` [config-recording]
+
+[![dynamic config](../images/dynamic-config.svg "") ](#dynamic-configuration)
+
+| Environment | Django/Flask | Default |
+| --- | --- | --- |
+| `ELASTIC_APM_RECORDING` | `RECORDING` | `true` |
+
+Enable or disable recording of events. If set to false, then the Python agent does not send any events to the Elastic APM server, and instrumentation overhead is minimized. The agent will continue to poll the server for configuration changes.
+
+
+## Logging Options [logging-options]
+
+
+### `log_level` [config-log_level]
+
+[![dynamic config](../images/dynamic-config.svg "") ](#dynamic-configuration)
+
+| Environment | Django/Flask | Default |
+| --- | --- | --- |
+| `ELASTIC_APM_LOG_LEVEL` | `LOG_LEVEL` |  |
+
+The `logging.logLevel` at which the `elasticapm` logger will log. The available options are:
+
+* `"off"` (sets `logging.logLevel` to 1000)
+* `"critical"`
+* `"error"`
+* `"warning"`
+* `"info"`
+* `"debug"`
+* `"trace"` (sets `logging.log_level` to 5)
+
+Options are case-insensitive
+
+Note that this option doesn’t do anything with logging handlers. In order for any logs to be visible, you must either configure a handler ([`logging.basicConfig`](https://docs.python.org/3/library/logging.md#logging.basicConfig) will do this for you) or set [`log_file`](#config-log_file). This will also override any log level your app has set for the `elasticapm` logger.
```

Comment:
Same issue with html translated to md in an external url, looks like all external links with .html in the url are broken